addons/controllers/clusterbootstrap_controller.go

```diff
@@ -67,6 +67,7 @@ type ClusterBootstrapReconciler struct {
 	aggregatedAPIResourcesClient client.Client
 	// helper for looking up api-resources and getting preferred versions
 	gvrHelper util.GVRHelper
+	cbHelper  *clusterbootstrapclone.Helper
```

Comment:
Lets not store this as a controller level variable that is long lived, if we need to we need to investigate for memory leaks in a scaled environment before we can merge this.

Reply:
I will take a look today at this one

Reply:
Fixed